lib/tapjoy/ldap/api/group.rb

```diff
@@ -0,0 +1,75 @@
+require 'memoist'
+module Tapjoy::LDAP::API
+  # API methods for managing LDAP Groups
+  module Group
+    class << self
+      extend Memoist
+      def create(group_name, group_type)
+        Tapjoy::LDAP.client.add(
+          distinguished_name(group_name),
+          ldap_attr(group_name, group_type)
+        )
+      end
+
+      def destroy(group_name)
+        Tapjoy::LDAP.client.delete(distinguished_name(group_name))
+      end
+
+      def update(group_name, username, operation)
+        Tapjoy::LDAP.client.modify(
+          distinguished_name(group_name),
+          [[operation, :memberUid, username]]
```

Comment:
That's what we were doing before: https://github.com/Tapjoy/ldap_tools/blob/master/lib/tapjoy/ldap/group/remove_user.rb#L30